components/automate-ui/src/app/pages/team-add-users/team-add-users.component.html

```diff
@@ -0,0 +1,50 @@
+<chef-page
+  subheading="Automate only displays local users."
+  [attr.secondary-btn-show]="showSecondary"
+  [attr.heading]="getHeading()"
+  [attr.confirm-btn-text]="getConfirmBtnText()"
+  [attr.page-loading]="loading"
+  [attr.confirm-loading]="addingUsers"
+  [attr.disable-confirm]="usersToAddValues().length === 0"
+  [attr.error-message]="getErrorMessage()"
+  (confirm)="addUsers()"
+  (close)="closePage()">
+  <div *ngIf="!loading">
+    <div id="no-users-container" *ngIf="usersNotFiltered()?.length === 0">
+      <h3>There are no more local <a [routerLink]="['/settings', 'users']" target="_blank">users</a> to add, create some more!</h3>
```

Comment:
```suggestion
      <h3>There are no more local <a [routerLink]="['/settings', 'users']" target="_blank">users</a> to add; create some more!</h3>
```

Reply:
@tylercloke could we make that h3 a paragraph please

Reply:
@susanev ^

Reply:
Lol github is trolling.